packages/react-core/src/components/Button/Button.tsx

```diff
@@ -69,29 +72,46 @@ const Button: React.FunctionComponent<ButtonProps & InjectedOuiaProps> = ({
   icon = null,
   ouiaContext = null,
   ouiaId = null,
-  tabIndex = null as number,
+  tabIndex = null,
   ...props
 }: ButtonProps & InjectedOuiaProps) => {
   const Component = component as any;
   const isButtonElement = Component === 'button';
+  const preventDefaultActions = {
+    onKeyPress: (event: Event) => {
```

Comment:
could we see this being modifiable from props? lots of events exist https://reactjs.org/docs/events.html.
if so it could take an array list of strings and then we can dynamically build out this object

Reply:
I like that idea, and we could just give it a sensible default. I wonder if the default should include anything more than `['onKeyPress', 'onClick']`, I expect most people won't bother overriding it.

Reply:
Good call! I [took a stab at it](https://github.com/patternfly/patternfly-react/pull/3801/commits/ad1c64b05402fff98f7598dc17eeb7bbda5b5760#diff-51ac3b180a71f3ee94153db042f8acd9R85-R86). It seems to be working as one would expect. Please let me know if you find any issues with this! I added a simple cypress test to verify this new prop `inoperableEvents` works - let me know if you can think of any other scenarios that would be good to verify.